script/configs/hem_config.yaml

```diff
@@ -0,0 +1,89 @@
+# ------------------------------------------------------------------------------
+# Data Selection
+# ------------------------------------------------------------------------------
+
+data_selection: 
+
+# Geography --------------------------------------------------------------------
+# Currently, geography is tied to ReEDS balancing areas (BAs). Valid values are
+# "p1", "p2", ..., "p134". This map shows where the BAs are located: 
+# https://www.nrel.gov/analysis/assets/pdfs/reeds-model-regions-map.pdf.
+
+# HEM analyses have been conducted for:
+#     Nevada: "p13", "p14"
+#     New England: "p129", "p130", "p131", "p132", "p133", "p134"
+
+  balancing_areas:
+    - "p129"
+    - "p130"
+    - "p131"
+    - "p132"
+    - "p133"
+    - "p134"
+
+# Time -------------------------------------------------------------------------
+
+  base_year: 2020
+  num_future_years: 2
+
+# Agent Resolution--------------------------------------------------------------
+
+  num_ipps: 1
+
+# Run Details ------------------------------------------------------------------
+
+# String used to distinguish the input data folder from others
+  folder_suffix: "20240806"
+
+# Parse options-----------------------------------------------------------------
+  
+  der_aggregator: true
+  set_nuclear_varcost_negative: true
+  load_profiles_subdir: "NewEngland"
+
+# Whether the input folder needs to be (re)created
+  skip_parse: false
+
```

Comment:
Let's just assume that we won't parse inputs as part of `hem.jl` and delete this section. 

It might also make sense to simplify the above to just a `runs` folder that defaults to HEMData.jl/runs and a input folder name.

Reply:
Deleted it.